src/store/reducers/currentUser.js

```diff
@@ -12,7 +12,8 @@ const currentUser = (state = initialState, action) => {
         token: action.payload.token,
         user_type: action.payload.user_type,
         name: action.payload.name,
-
+        hotel_name: action.payload.hotel_name,
```

Comment:
Not sure if you've changed this already, but in this file, we also need to set '_id: action.payload.id,' instead of '_id: action.payload._id'

Reply:
Should be good now. I removed the _

Reply:
Awesome, one more, can we add the motto onto currentUser?

Reply:
Updated to include motto